src/logging.js

```diff
@@ -133,14 +133,42 @@ export async function makeRequestMiddleware(
 ): Promise<Middleware> {
     // This is the logger that captures requests handled by our express server.
     return args.dev
-        ? (req: $Request, res: $Response, next: NextFunction) => {
-              // $FlowIgnore: We make this up for google
-              req.log = logger;
-              next();
-          }
-        : await lw.express.makeMiddleware(logger);
+        ? /**
+           * If we're in dev, we're going to use the expressWinston logger
+           */
+          Promise.resolve(
+              expressWinston.logger({
+                  /**
+                   * Specify the level that this logger logs at.
+                   * (use a function to dynamically change level based on req and res)
+                   *     `function(req, res) { return String; }`
+                   */
+                  level: "info",
+
+                  /**
+                   * Use the logger we already set up.
+                   */
+                  winstonInstance: logger,
+                  expressFormat: true,
+                  colorize: true,
+                  meta: false,
+              }),
+          )
+        : /**
+           * Otherwise, we're using the Google middleware
+           */
+          await lw.express.makeMiddleware(logger);
```

Comment:
To clarify: we're awaiting here - is the result of the await also a promise? Considering that we're returning a promise if we're not in dev it seems like the `await` is unnecessary (since it would just return a promise either way).

Reply:
Yeah, there should be a linter to catch this. We don't need the `await` here for sure.